### DISCUSSION TITLE
Can't Install

### DISCUSSION THREAD

**Discussion by MisterEnigma** [bridge-core/bridge.#284] [Q&A]
**Summary**
For some reason I cant download any of the recent versions of bridge, I left it installing for hours and it still didn't complete and it left no errors?

**To Reproduce**
1. Go to execute the bridge web-setup
2. press "run"

**Observed behavior**
It shows it installing but it only stays a quarter filled
**Expected behavior**
Showing full green bar showing it completed installing

**Platform (please complete the following information):**
 - OS: [e.g. Windows 7]
 - App Version: [e.g. 1.7.9]

**Comment by solvedDev** [comment]
The installation needs an active internet connection and the time it takes depends on your internet speed. Try closing the installer & running it again :)

**Comment by MisterEnigma** [comment]
> 
> 
> The installation needs an active internet connection and the time it takes depends on your internet speed. Try closing the installer & running it again :)

See thats the thing I done it hundreds of times and even tried 2 different networks to connect too and it still didnt work, but I'll keep trying

**Comment by ItsCiaPro** [comment]
U need windows 10 bc it only works if u have Minecraft Windows 10 Edition installed
